<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Takiyasha GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="title">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Takiyasha GUI</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="input_group_box">
        <property name="title">
         <string>输入</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>文件列表:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="files_list_view">
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="open_files">
             <property name="text">
              <string>打开文件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="open_folder">
             <property name="text">
              <string>打开文件夹</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="options_group_box">
        <property name="title">
         <string>选项</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="no_parallel">
           <property name="text">
            <string>禁用多线程</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="test_only">
           <property name="text">
            <string>仅检测</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="try_fallback">
           <property name="text">
            <string>启用后备算法</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="no_tag">
           <property name="text">
            <string>不写入元数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="overwrite">
           <property name="text">
            <string>覆盖已存在的文件</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="avoid_searching_tag">
           <property name="text">
            <string>禁止从网络下载元数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="open_dist_folder">
           <property name="text">
            <string>打开输出文件夹</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>最大线程数:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="max_thread_count">
             <property name="maximum">
              <number>65535</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="output_group_box">
      <property name="title">
       <string>输出</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>输出路径:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="output_path"/>
         </item>
         <item>
          <widget class="QToolButton" name="select_output_path">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="use_source_path">
         <property name="text">
          <string>输出到源目录</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="start">
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>信息</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTextBrowser" name="message_box"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
      <property name="format">
       <string>%v of %m</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>538</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout_TakiyashaGUI"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen_Files"/>
    <addaction name="actionOpen_Folder"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout_TakiyashaGUI">
   <property name="text">
    <string>关于Takiyasha GUI</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="actionOpen_Files">
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>files_list_view</tabstop>
  <tabstop>open_files</tabstop>
  <tabstop>open_folder</tabstop>
  <tabstop>output_path</tabstop>
  <tabstop>select_output_path</tabstop>
  <tabstop>use_source_path</tabstop>
  <tabstop>no_parallel</tabstop>
  <tabstop>test_only</tabstop>
  <tabstop>try_fallback</tabstop>
  <tabstop>no_tag</tabstop>
  <tabstop>avoid_searching_tag</tabstop>
  <tabstop>start</tabstop>
  <tabstop>message_box</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionAbout_Qt</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>about_qt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout_TakiyashaGUI</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>about_takiyasha_gui()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_files</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_files()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_folder</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_folder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Files</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_files()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Folder</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_folder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>514</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_source_path</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>use_source_path_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_output_path</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>select_output_path()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open_files()</slot>
  <slot>open_folder()</slot>
  <slot>select_output_path()</slot>
  <slot>use_source_path_changed(int)</slot>
  <slot>start()</slot>
  <slot>about_qt()</slot>
  <slot>about_takiyasha_gui()</slot>
  <slot>exit()</slot>
 </slots>
</ui>
